tools/jsdoc/jsdoc_extractor/.gitignore

```diff
@@ -0,0 +1,74 @@
+# This file is used to ignore files which are generated
```

Comment:
Could the generated files go in a subdirectory, and then we ignore that directory?